modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -246,11 +259,20 @@ private String _addOrReplaceParameters(
 			StringUtil.merge(
 				interpolatedNewParameterNames, StringPool.COMMA_AND_SPACE));
 
-		sb.append(StringPool.CLOSE_PARENTHESIS);
+		sb.append(lastCharacter);
 
 		return sb.toString();
 	}
 
+	private String _addOrReplaceTypeParameters(
+		String newMethodCall, List<String> newTypeParameterNames,
+		List<String> typeParameterNames) {
+
+		return _addOrReplaceParameters(
+			StringPool.GREATER_THAN, newMethodCall, newTypeParameterNames,
+			typeParameterNames, "generic#");
```

Comment:
@TamyrisTorres, to keep consistency, let's rename `generic#` to `typeParam#`

Reply:
resolved!